File: MovieTheater/MovieTheater.Domain/Entities/Screening.cs
namespace MovieTheater.Domain.Entities;

public class Screening
{
    public Guid Id { get; set; }
    public DateTime Start { get; set; }
    public decimal Price { get; set; }

    public virtual Movie Movie { get; set; }
    public Guid MovieId { get; set; }

    public virtual Hall Hall { get; set; }
    public Guid HallId { get; set; }
    
    public Guid? MovieLanguageId { get; set; }
    public virtual MovieLanguage? MovieLanguage { get; set; }

    public virtual ICollection<Reservation> Reservations { get; set; }
}

File: MovieTheaterUI/src/app/shared/models/screening.ts
export interface IScreeningModel{
  id: string;
  hallId: string;
  movieLanguageId: string;
  movieId: string;
  movieTitle: string;
  price: number;
  start: Date;
  reservations: string[];
}

export interface ICreateScreeningModel{
  hallId: string;
  movieId: string;
  price: number;
  start: Date;
}
